### DISCUSSION TITLE
Propeller test on M3 and M4 [FAIL]

### DISCUSSION THREAD

**Discussion by iNavin** [bitcraze/discussions#953] [Q&A] [answered]
```
HEALTH: Acc noise floor variance X+Y:0.009472, (Z:0.004127)
HEALTH: Motor M1 variance X+Y: 0.50 (Z:0.36), voltage sag:0.33
HEALTH: Motor M2 variance X+Y: 1.37 (Z:1.11), voltage sag:0.32
HEALTH: Motor M3 variance X+Y: 18.20 (Z:7.03), voltage sag:0.32
HEALTH: Motor M4 variance X+Y: 11.83 (Z:22.69), voltage sag:0.33
HEALTH: Propeller test on M3 [FAIL]. low: 0.0, high: 2.50, measured: 18.20
HEALTH: Propeller test on M4 [FAIL]. low: 0.0, high: 2.50, measured: 11.83
ESTKALMAN: WARNING: Kalman prediction rate low (83)
ESTKALMAN: State out of bounds, resetting
ESTKALMAN: State out of bounds, resetting

```
I am new crazyflie, i got this logs on propellor test -  what does this mean?

**Comment by knmcguire** [comment, marked as answer]
Hi! It means that the propeller on the M3 and M4 motors are unbalanced and giving an high vibration during the test. If you want to improve your flight performance, you should either replace those motors with a new one or balance them with these instructions: https://www.bitcraze.io/documentation/tutorials/balancing-propellers/

Mind that these tests can also fail due to the motor, so also double check if the shaft of the motor is not bent. If replacing the propellers on that particular motor doesn't work, then you can also try to replace the full motor itself and see if that improves things.

**Reply by iNavin** [reply]
Hi knmcguire,
Thankyou for your answer after changing the propellers the error is not being shown, but now i am seeing this propeller test logs :
HHEALTH: Acc noise floor variance X+Y:0.006361, (Z:0.004203)
HEALTH: Motor M1 variance X+Y: 1.15 (Z:0.67), voltage sag:0.40
HEALTH: Motor M2 variance X+Y: 1.49 (Z:3.68), voltage sag:0.40
HEALTH: Motor M3 variance X+Y: 2.01 (Z:4.29), voltage sag:0.40
HEALTH: Motor M4 variance X+Y: 0.34 (Z:0.80), voltage sag:0.42
STAB: WARNING: stabilizer loop rate is off (537)
ESTKALMAN: WARNING: Kalman prediction rate low (3)
ESTKALMAN: State out of bounds, resetting
SUP: Can not fly
SUP: Ready to fly

what does the last 5 lines mean ?

**Reply by knmcguire** [reply]
- STAB: is just warning that the [stabilizer loop](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/sensor-to-control/) (the one that handle controls, estimators and such) is not running at a 1000 hz. But that seems to have sort itself out.
- ESTKALMAN: Those are just general warnings of the state estimator. It usually needs time to converge to the right position
- SUP: that is the supervisor states, that is waiting first for the crazyflie to fully initialize and once everything is ready, it gives the final OK. 

For any new question, start a new discussion, as this was technically not about the initial question anymore (and you have selected an answer already)
